recipe/meta.yaml

```diff
@@ -19,13 +19,15 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.7
     - pip
     - setuptools
 
   run:
-    - python >=3.6
-    - click >=7.1.2
+    - python >=3.7
+    - click >=8.0
+    # should be py<310 but we want to keep this noarch: python
+    - importlib-metadata
```

Comment:
@conda-forge/flask, are you all okay with this?

Reply:
I'm okay with this :)

Reply:
Looking into this further, I don't see any evidence that it does any harm to use the external package for py>=310 so I think I'll merge.  Thanks for your input @nehaljwani.